.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
please remove these vs-code files :) It would be awesome, if you could add a `.gitignore` part for vscode. You can probably find something useful here [](https://github.com/github/gitignore)

Reply:
(same goes for the .vs/slnx.sqlite file)